images/runtime/php-fpm/7.4/base.bullseye.Dockerfile

```diff
@@ -93,7 +90,7 @@ RUN set -eux; \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* ]]; then \
+    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.* ]]; then \
```

Comment:
Same comment as above. Also, don't we want to make these changes in [the template.base dockerfiles](https://github.com/microsoft/Oryx/blob/main/images/runtime/php/template.base.Dockerfile#L103-L112) as well? Also in the [php-fpm template.base](https://github.com/microsoft/Oryx/blob/main/images/runtime/php-fpm/template.base.Dockerfile#L89-L97) dockerfiles?